### PR TITLE
Add dryrun flag

### LINKED ISSUE #8: Add dryrun flag
Add a dryrun option so no changes are really commited but information about potential changes is displayed.

### PR DESCRIPTION
**Pull request by miguelbernadi**
Close #8

### CODE REVIEW

**Comment by miguelbernadi** [comment]
[B] Until PR #7 is merged in, as this one includes its changes.

File: mcc/ttl/main.go [hunk 1]
```diff
@@ -132,11 +133,13 @@ func UpsertResourceRecordSetTTL(
 		log.Panic(err.Error())
 	}
 	// Submit batch changes
-	changeResponse, err = svc.ChangeResourceRecordSets(changeRRSInput)
+	if !dryrun {
+		changeResponse, err = svc.ChangeResourceRecordSets(changeRRSInput)
```
Comment (ifosch) [inline mcc/ttl/main.go:138]: Shouldn't be `changeResponse` getting some content back to return from this function?

**Comment by ifosch** [comment]
Probably `dryrun` should be used in the evaluation to run `WaitForChangeToComplete` as well.

**Comment by ifosch** [comment]
Sorry, I closed this by mistake.

**Comment by ifosch** [comment]
For reference on following PRs: I think we would prefer to have changes like the Codeclimate change and the minor fix for the error on separate PRs ;)

**Comment by miguelbernadi** [comment]
Fixed conflict with master branch after merging #7

File: mcc/ttl/main.go (hunk 1 above)
Comment (miguelbernadi) [reply mcc/ttl/main.go:138]: Right now, if it's a dryrun run `changeResponse` is nil. As the return value from the function isn't used afterwards before the end of the program (if dryrun there is no execution of `WaitForChangeToComplete`) I think it's fine for now. Nonetheless it provides dangerous side-effects if we try to use it in the future so we will need to fix/improve this behaviour before extending the tool further.

**Comment by miguelbernadi** [comment]
The missing change was on my local machine but I had forgotten to commit it. I rebased the PR to make it fit into the same commit.

**Comment by ifosch** [comment]
+1 Perfect!

**Pull request merged**